authzed/api/v1/watch_service.proto

```diff
@@ -72,4 +82,11 @@ message WatchResponse {
   // given to SpiceDB during the transaction that produced the changes in this response.
   // This field may not exist if no transaction metadata was provided.
   google.protobuf.Struct optional_transaction_metadata = 3;
+
+  // schema_updates are the schema update events that have occurred since the
+  // last watch response.
+  repeated ReflectionSchemaDiff schema_updates = 4;
+
+  // is_checkpoint, if true, indicates that a revision was reached.
```

Comment:
@miparnisari I think we should clarify this: "Reaching a revision" does not tell the client what guarantees they get from a checkpoint.

I would frame it as "a checkpoint indicates the server guarantees the client will not observe any changes at a revision below or equal to the revision in this response"

Reply:
Sounds good! Added to https://github.com/authzed/api/pull/135